src/commands/GitHubCommand.ts

```diff
@@ -0,0 +1,52 @@
+import { Message, MessageEmbed } from "discord.js";
+import Command from "../abstracts/Command";
+import GitHubService from "../services/GitHubService";
+import { EMBED_COLOURS } from "../config.json";
+
+class GitHubCommand extends Command {
+	constructor() {
+		super(
+			"github",
+			"Shows the repo of the given user."
+		);
+	}
+
+	async run(message: Message, args: string[]) {
+		const embed = new MessageEmbed();
+
+		if (!args || typeof args[0] === "undefined") {
+			embed.setTitle("Error");
+			embed.setDescription("You must provide a username and repo from GitHub.");
+			embed.addField("Correct Usage", "?github <username>/<repository>");
+			embed.setColor(EMBED_COLOURS.ERROR);
+
+			await message.channel.send({ embed });
+		} else {
+			const [user, repoName] = args[0].split("/");
+
+			try {
+				const GitHubRepo = GitHubService.getInstance();
```

Comment:
This variable's name isn't true - can you rename it to `GitHub` or `GitHubService` please.